go/client/cmd_simplefs_sync_show.go

```diff
@@ -53,6 +53,7 @@ func (c *CmdSimpleFSSyncShow) Run() error {
 		ui.Printf("Syncing disabled\n")
 	case keybase1.FolderSyncMode_ENABLED:
 		ui.Printf("Syncing enabled\n")
+		ui.Printf("Prefetch Status: %s\n", res.Status.PrefetchStatus)
```

Comment:
I'd prefer if this was a little more human friendly, something like:

```go
switch res.Status.PrefetchStatus {
  case keybase1.PrefetchStatus_COMPLETE:
    ui.Printf("Status: fully synced\n")
  case keybase1.PrefetchStatus_IN_PROGRESS:
    ui.Printf("Status: sync in progress\n") // TODO: add progress stats here
  case keybase1.PrefetchStatus_NOT_STARTED:
    ui.Printf("Status: sync not yet started\n")
  default:
    ui.Printf("Status: unknown\n")
}
```

Reply:
(Of course you can massage that a bit so that the "Status:...\n" part isn't repeated everywhere.